plugins/misp/plugin.spec.yaml

```diff
@@ -3,17 +3,45 @@ extension: plugin
 products: [insightconnect]
 name: misp
 title: MISP
-description: MISP is an open source threat sharing platform
-version: 5.0.3
+description: "[MISP](http://www.misp-project.org/) is an open source threat sharing platform. Gather, store and then find correlations of indicators of compromise. Quality of data is determined by the open source community. This plugin utilizes the [MISP API](https://circl.lu/doc/misp/automation/index.html) and leverages the [pymisp](https://github.com/CIRCL/PyMISP) library"
+version: 6.0.0
 connection_version: 5
```

Comment:
does connection_version need bumped on a major release?

Reply:
no the connection version will only need bumped if there is a change to the connection object 

eg went from taking a name and pass to now using an api key

Reply:
ahh ok, good to know!